src/main/java/seedu/address/logic/parser/AddModuleCommandParser.java

```diff
@@ -0,0 +1,44 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_MODULE_CODE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_MODULE_TITLE;
+import static seedu.address.logic.parser.ParserUtil.arePrefixesPresent;
+import static seedu.address.model.module.Module.EMPTY_MODULE_TITLE;
+
+import seedu.address.logic.commands.AddModuleCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.module.Module;
+import seedu.address.model.module.ModuleCode;
+import seedu.address.model.module.ModuleTitle;
+
+/**
+ * Parses input arguments and creates a new AddCommand object
+ */
+public class AddModuleCommandParser implements Parser<AddModuleCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the AddCommand
+     * and returns an AddCommand object for execution.
```

Comment:
```suggestion
     * and returns an {@code AddModuleCommand}object for execution.
```
_see previous comment_